packages/lib-classifier/src/plugins/tasks/components/TaskInput/TaskInput.spec.js

```diff
@@ -11,54 +14,69 @@ const radioTypeAnnotation = {
 }
 
 describe('TaskInput', function () {
-  describe('render', function () {
-    let wrapper
-    before(function () {
-      wrapper = shallow(<TaskInput annotation={radioTypeAnnotation} index={0} type='radio' />)
-    })
-
-    it('should render without crashing', function () {
-      expect(wrapper).to.be.ok()
-    })
+  function withGrommet() {
+    return function Wrapper({ children }) {
+      return (
+        <Grommet theme={zooTheme}>
+          {children}
+        </Grommet>
+      )
+    }
+  }
```

Comment:
I think this is the last spec file you can wrap around the `TaskInput.stories.js` and get rid of the `Grommet` dependency in the spec file.